style.css

```diff
@@ -382,7 +382,7 @@ span {
   color: #172b4d;
 }
 
-form {
+.contact-details {
```

Comment:
- [ ] Your contact section is well designed, good job :+1:  But please make it match with the Figma template by using the same size for your input and also the same border color for your textarea  because as a developer it is very important to pay attention to details :smile: 

#### Figma design
![Screenshot from 2022-08-30 18-11-50](https://user-images.githubusercontent.com/6615394/187512832-42fd3164-89a1-4aed-b903-601e988a4369.png)

#### Your design
![Screenshot from 2022-08-30 18-09-09](https://user-images.githubusercontent.com/6615394/187512840-03ef8694-6d2c-4c80-b4a7-96ef34ed3262.png)